File: VB/ServerSide/Controllers/QueryBuilderController.vb
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc.Controllers

Namespace ServerSide.Controllers
	Public Class QueryBuilderController
		Inherits QueryBuilderApiController

		Public Overrides Function Invoke() As ActionResult
			Dim result = MyBase.Invoke()
			' Allow cross-domain requests.
			Response.AppendHeader("Access-Control-Allow-Origin", "*")
			Return result
		End Function
	End Class
End Namespace

File: CS/ServerSide/Controllers/QueryBuilderController.cs
using System.Web.Mvc;
using DevExpress.Web.Mvc.Controllers;

namespace ServerSide.Controllers {
    public class QueryBuilderController : QueryBuilderApiControllerBase {
        public override ActionResult Invoke() {
            var result = base.Invoke();
            // Allow cross-domain requests.
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }
    }
}